code/modules/assembly/signaler.dm

```diff
@@ -63,7 +63,7 @@
 /obj/item/device/assembly/signaler/activate()
 	if(cooldown > 0)
 		return 0
-	cooldown = 2
+	cooldown = 0.5
```

Comment:
That's just a second.

Reply:
it does support negative values and i updated the pr title/desc